openpype/settings/defaults/project_settings/hiero.json

```
@@ -10,15 +10,15 @@
             "rules": {}
         },
         "workfile": {
-            "ocioConfigName": "nuke-default",
-            "workingSpace": "linear",
-            "sixteenBitLut": "sRGB",
-            "eightBitLut": "sRGB",
-            "floatLut": "linear",
-            "logLut": "Cineon",
-            "viewerLut": "sRGB",
-            "thumbnailLut": "sRGB",
-            "monitorOutLut": "sRGB"
+            "ocioConfigName": "aces_1.2",
+            "workingSpace": "scene_linear (ACES - ACEScg)",
+            "viewerLut": "ACES/sRGB",
+            "thumbnailLut": "ACES/sRGB",
```

```suggestion
            "thumbnailLut": "sRGB (ACES)",
```
I do realize that this might seem to be unnecessary step but since the value needs to be set as `ACES/sRGB` . Just a suggestion, since the PR is stating that settings should be aligned. 

Functionality for string conversion can be found here https://github.com/ynput/ayon-core/blob/enhancement/AY-974_Nuke-simplified-deadline-submission-on-write-node/client/ayon_core/hosts/nuke/api/lib.py#L2855

---

I think since the code is already in Nuke, we should align Nuke with Hiero rather than the other way around.